rocAL/rocAL_pybind/amd/rocal/types.py

```diff
@@ -58,6 +58,8 @@
 #     RocalTensorLayout
 from rocal_pybind.types import NHWC
 from rocal_pybind.types import NCHW
+from rocal_pybind.types import NFHWC
+from rocal_pybind.types import NFCHW
```

Comment:
Why can't we import entire rocal_pybind.types? instead of importing one by one
Does it has too many types?